http/src/v0/root_files.rs

```diff
@@ -117,26 +102,35 @@ pub fn get<T: IpfsTypes>(
 async fn get_inner<T: IpfsTypes>(ipfs: Ipfs<T>, args: GetArgs) -> Result<impl Reply, Rejection> {
     use futures::stream::TryStreamExt;
 
-    let path = IpfsPath::try_from(args.arg.as_str()).map_err(StringError::from)?;
+    let path = args.arg.into_inner();
 
     // FIXME: this is here until we have IpfsPath back at ipfs
```

Comment:
is this FIXME not void now?

Reply:
```suggestion
```

almost got it when cleaned up the cat comments :)